utils/R/utils.R

```diff
@@ -609,18 +613,26 @@ misc.convert <- function(x, u1, u2) {
 
 ##' function to check whether units are convertible by misc.convert function
 ##' @title misc.are.convertible
+##' @export
 ##' @param u1 unit to be converted from, character
 ##' @param u2 unit to be converted to, character
 ##' @return logical
-##' @export
-##' @author Istem Fer
+##' @author Istem Fer, Shawn Serbin
+#misc.are.convertible <- function(u1, u2) {
+#  if (match(u1, c("umol C m-2 s-1", "kg C m-2 s-1")) ==
+#      match(u2, c("kg C m-2 s-1", "umol C m-2 s-1"))) {
+#    return(TRUE)
+#  } else {
+#    return(FALSE)
+#  }
+#}
 misc.are.convertible <- function(u1, u2) {
-  if (match(u1, c("umol C m-2 s-1", "kg C m-2 s-1")) ==
-      match(u2, c("kg C m-2 s-1", "umol C m-2 s-1"))) {
-    return(TRUE)
-  } else {
-    return(FALSE)
-  }
+  ifelse(match(u1, c("umol C m-2 s-1", "kg C m-2 s-1")) ==
```

Comment:
@istfer if you think using ifelse() here is OK (revised version) I will delete the commented out code. I left that there for comparison.  

I will edit this before we merge these changes